aopy/preproc/quality.py

```diff
@@ -1,14 +1,46 @@
 import copy
 import matplotlib
-from .. import precondition
-from .. import analysis
-from ..utils import print_progress_bar
-from ..visualization.base import plot_image_by_time
-
 import numpy as np
 import numpy.linalg as npla
 import scipy.signal as sps
 import matplotlib.pyplot as plt
+import traceback
+from tqdm.auto import tqdm
+
+from .. import precondition
+from .. import analysis
+from ..utils import print_progress_bar
+from ..visualization.base import plot_image_by_time
+from ..data.base import load_preproc_exp_data
+
+def detect_buggy_entries(preproc_dir, subjects, ids, dates):
```

Comment:
what is an entry, and what does it mean to be buggy? 
I propose we name this `detect_bad_exp_data`, and can you include a docstring that explains the types of errors that make something bad?